packages/components/src/components/pagination/_pagination.scss

```
@@ -42,6 +42,19 @@ $css--helpers: true;
     @include carbon--breakpoint('md') {
       overflow: initial;
     }
+
+    // mobile friendly pagination
+    @include carbon--breakpoint-down('md') {
+      .#{$prefix}--pagination__left > *,
+      .#{$prefix}--pagination__right > * {
+        display: none;
+      }
+
+      .#{$prefix}--pagination__items-count,
+      .#{$prefix}--pagination__controll-buttons {
```

Actually I think the above is fixed with this change: 

```suggestion
      .#{$prefix}--pagination__control-buttons {
```